packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -117,8 +117,9 @@ function Cell(props) {
       event.stopPropagation();
       event.preventDefault();
     } else if (onCellSelect) {
-      onCellSelect({ rowId, columnId }, { row: rowIndex, col: columnIndex });
+      onCellSelect({ rowId, columnId }, { row: rowIndex, col: columnIndex }, !!event.shiftKey, false);
```

Comment:
What is the last argument passed to onCellSelect?  Its purpose is not clear.

Reply:
The anchor row for selecting multiple rows differs based on the selection made with click or space key. The purpose of the last argument is to recognize if the selection was initiated by keyboard.